asset/tx.go

```diff
@@ -100,7 +100,11 @@ func VirtualGenesisTxIn(newAsset *Asset) (*wire.TxIn, mssmt.Tree, error) {
 
 	prevOut := VirtualTxInPrevOut(treeRoot)
 
-	return wire.NewTxIn(prevOut, nil, nil), inputTree, nil
+	txIns := []*wire.TxIn{
```

Comment:
I don't think this needs to change. There's only ever a single virtual txin still.